docs/xray/auditnpm/help.go

```diff
@@ -0,0 +1,5 @@
+package auditnpm
+
+const Description = "Execute an audit npm command, using the configured Xray details."
```

Comment:
Execute an audit npm command, using the configured Xray details
-->
Scan a VCS project with JFrog Xray, after the sources were cloned locally.